client/my-sites/woocommerce/woop/style.scss

```diff
@@ -1,3 +1,17 @@
 body.is-section-woocommerce-installation.theme-default.color-scheme {
 	--color-surface-backdrop: var( --studio-white );
 }
+
+.woocommerce_landing-page {
+	.empty-content__title {
+		font-family: Recoleta;
```

Comment:
This feels like something that should be done in the `EmptyContent` component directly but would widen testing considerably. (see "Inbox" page at the moment for why I think it might be worth doing)

Reply:
It looks like it's used pretty widely. 😬 I think we're okay keeping it here for now, but I see what you mean about the Inbox page, where it's missing that font.

Reply:
I am with @allilevine in this one this will widen testing a lot, I also feel we can totally add some focused personalization tho the base component through css per page like we are doing right now